src/components/content/Content.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import CreateTask from "src/components/tasks/CreateTask";
+import TaskItems from "src/components/tasks/TaskList";
+import { addTasks } from "src/store/actions/taskActions";
+import { Button } from "src/components/button/Button";
+import {
+  ALT_ADD_TEXT,
+  CLASS_BUTTON_CREATE,
+  ICON_ADD,
+  TEXT_CREATE,
+} from "src/utils/constants/constants";
+import "src/components/content/Content.component.scss";
+
+const Content = () => {
+  const [showTask, setShowTask] = useState(false);
+  const dispatch = useDispatch();
+
+  function onAddTask(sanitizedValue) {
+    setShowTask(!showTask);
+    dispatch(addTasks(sanitizedValue));
+  }
+  function showInputField() {
+    showTask || setShowTask(!showTask);
+  }
+  function hideInputField() {
+    showTask && setShowTask(!showTask);
+  }
```

Comment:
Look like this functions are similar with different behaviour, what about combine this two function and handle from one. Can you optimise the implementation here? Thanks!